app/posts/find-teacher-training/2021-08-06-changes-to-find-at-the-end-of-2021-cycle.md

```diff
@@ -0,0 +1,114 @@
+
+To prevent candidates applying for the first time from adding courses starting in the 2021 to 2022 academic year (which they're no longer eligible to apply to) we will not take them through the usual "You selected a course" journey:
+
+![Candidates applying for the first time after the deadline cannot select a course.](select-course-journey.png)
+
+The candidate is instead sent straight to their task list where they’re informed that [courses are closed](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-find-reopens%2C-they-cannot-find-courses) and that [they cannot submit yet](/apply-for-teacher-training/end-of-cycle-2021/#if-they-carry-over-before-apply-reopens%2C-they-cannot-submit).
+
+Candidates applying again after the deadline to apply for the first time will go through the course selection journey as normal.
+
+## Candidates are discouraged from searching for courses after the deadline to apply again has passed
+
+Although candidates cannot submit an application after the [deadline to apply again](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/#21-september-2021-at-6pm) pas passed, we continue to show course listings so that providers and candidates who’ve already applied can finalise any post-submission activity (such as making and accepting offers).
+
+A banner lets candidates know that courses are closed but that they can start or continue applying for the next cycle anyway:
+
+![The deadline to apply again has passed.](courses-closed-banner.png)
+
+We also remove the [‘Apply for this course’ button](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/#if-a-candidate-applying-for-the-first-time-tries-to-select-a-course-after-the-deadline) after the deadline to apply again because it implies that you're applying for a particular course, when in fact there's no guarantee that the 2021 to 2022 course listings will be available the following cycle.
+
+The following message replaces the button, enabling candidates to start or continue an application despite courses being closed:
+
+![Stop allowing candidates to click 'Apply for this course'.](courses-closed-start-application-anyway.png)
+
+As before, candidates do not go through the ['you selected a course' journey](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/select-course-journey.png).
+
+## Candidates can no longer view courses when the recruitment cycle has ended
+
+[When the recruitment cycle ends](/find-teacher-training/changes-to-find-at-the-end-of-2021-cycle/#4-october-2021-at-midnight) candidates can no longer be placed on courses starting in the 2021 to 2022 academic year, so we stop showing the 2021 to 2022 courses.
+
+Anyone trying to visit Find will be informed that courses are closed, but we enable them start or continue an application anyway:
```

Comment:
Typo, missing 'to': 'but we enable them [to] start'. Or you could write 'but we let them start' instead.

Reply:
fanks!